.circleci/config.yml

```diff
@@ -69,7 +69,7 @@ jobs:
 
   "lint":
     docker:
-      - image: golangci/golangci-lint:v1.12
+      - image: golangci/golangci-lint:latest
```

Comment:
Nice catch, but don't use `latest` label, as we will have to track it and make sure we have everything (like go version) up to date.

Reply:
I think the linter should be on latest to quickly tell us new problem that have been discovered.
But I agree that it could be boring if suddenly the lint break for some code that is not related to the PR in progress.